makefile

```diff
@@ -378,6 +378,23 @@ phony += clean
 gqlgen:
 	go run github.com/99designs/gqlgen generate
 
+swaggo: 
+	make swaggo-build
+	make swaggo-render
+
+swaggo-install:
+	go install github.com/swaggo/swag/cmd/swag@latest
+
+swaggo-format:
+	swag fmt -g service/service.go
+
+swaggo-build:
+	swag init -g service/service.go
```

Comment:
Seems like `swag init --output` can specify a directory to put the docs in rather than having to `mv` it. Can we use that instead?